test/models_test.py

```diff
@@ -532,31 +532,28 @@ def test_repr(self, queue):
 class TestPrincipal(object):
     @pytest.fixture
     def principal(self):
-        return Principal(username="admin", roles=["bg-admin"], permissions=["bg-all"])
+        return Principal(username="admin", roles=None, permissions=["bg-all"])
```

Comment:
Is there a reason to take this out? I feel like we should keep this as part of our fixtures.